### ISSUE TITLE
The predicted values and the actual values differ by a constant.

### ISSUE THREAD

**Issue by HuaitaoZhu** [neuraloperator/neuraloperator#399]
Hello,

I am training with the provided FNO2d example using my own data. During training, although the 2D heatmaps for the predicted and actual values look the same, their actual numerical values differ by a constant. How can I solve this issue? Could it be a problem with the loss function or something else?

Thank you.
![output](https://github.com/user-attachments/assets/22e97ea5-a163-4b62-98d8-43ab3f371433)
![output2](https://github.com/user-attachments/assets/dc82840f-bfad-4e32-a393-1ede1015cfe3)

**Comment by dhpitt** [comment]
Hi @HuaitaoZhu, how are you handling data normalization/unnormalization?

**Comment by HuaitaoZhu** [comment]
My original intention was to predict a physical field with equal absolute values. The data is not normalized, and I made modifications to the original **load_darcy_flow_small** function, using randomly generated data instead：
x_train = torch.randn(n_train, 1, 2, 16)
y_train = x_train ** 2 + 2
x_test = torch.randn(n_test, 1, 2, [16)]
y_test = x_test ** 2 + 2

**Comment by HuaitaoZhu** [comment]
I attached my code, and I didn't make many modifications to the original code

https://github.com/HuaitaoZhu/temporary/blob/7b4f403664110cc85af48634597ddc5ad5ae343c/FNO_jupyter.ipynb
https://github.com/HuaitaoZhu/temporary/blob/7b4f403664110cc85af48634597ddc5ad5ae343c/data_generation.py

**Comment by HuaitaoZhu** [comment]
I commented out the code for normalizing y. Now, the absolute values of the predicted values are equal to the true values.
#y_train = output_encoder.transform(y_train)
![output3](https://github.com/user-attachments/assets/29edd81d-830e-4541-82bd-ca0f2a1850ad)

**Comment by dhpitt** [comment]
How recently did you pull the code from main? I recently pushed a fix to
output normalization.

**Comment by dhpitt** [comment]
Check code after #397 and #403

**Comment by HuaitaoZhu** [comment]
I read your comments, I found that I am still using the basic version 0.3.0 code(Dec 9, 2023), so far you have made a lot of changes in the code. I tried to clone directly and install the latest version, but I had some problems trying to load data on the new version, and I haven't seen the final results yet. I am trying to solve it, thank you very much for your help

**Comment by HuaitaoZhu** [comment]
Thanks for your help, the problem has been solved

**Comment by HuaitaoZhu** [comment]
Thanks for your help, the problem has been solved
